manifest.go

```diff
@@ -445,10 +445,13 @@ func applyManifestChange(build *Manifest, tc *pb.ManifestChange) error {
 	case pb.ManifestChange_DELETE:
 		tm, ok := build.Tables[tc.Id]
 		if !ok {
-			return fmt.Errorf("MANIFEST removes non-existing table %d", tc.Id)
+			for _, level := range build.Levels {
```

Comment:
Could we add a log here for FYI?

Reply:
@mangalaman93 of course, I would have done this but was a bit unsure of how you use the logger in badger.  Looks like all current warnings are thrown from an options object, should i cange the function signatures to pass the options object down to this function?

Reply:
yes, that seems alright I think.

Reply:
Ok I had an attempt, ive added it as a commit on top so you can check it separately, had to do some plumbing to get the options object in the right places. The most questionable chenge there is in manifest_test.go where i had to init a db object, hopefully that is correct?

Reply:
@harshil-goel could you take a look at this as well. Looks fine to me.